Sources/TootSDK/TootClient/TootClient+Lists.swift

```diff
@@ -14,6 +14,16 @@ public extension TootClient {
 
         return try await fetch([List].self, req)
     }
+    
+    /// User lists that you have added this account to.
+    func getLists(id: String) async throws -> [List] {
```

Comment:
This name doesn't fit to what the function does. Maybe try something like these to make it clear?

```suggestion
    func getContainingLists(accountId: String) async throws -> [List] {
```

```suggestion
    func getListsContainingAccount(id: String) async throws -> [List] {
```

Reply:
Oh I totally missed that, you're right!

Reply:
PR https://github.com/TootSDK/TootSDK/pull/186